CHANGELOG.md

```diff
@@ -9,7 +9,10 @@
 * Added support for tvOS and watchOS in images, fonts and color templates.  
 [Tom Baranes](https://github.com/tbaranes),
   [#145](https://github.com/AliSoftware/SwiftGen/pull/145)
-  
+* Added support for OSX in storyboards.  
```

Comment:
We should probably mention here that `class` has been renamed `customClass` and the Stencil context for storyboards has changed a bit. This would be especially important for people that potentially implemented their own custom templates for Storyboards and used the `class` entry in them.